src/pages/proposal-commons/treasury/treasury.tsx

```diff
@@ -72,9 +72,12 @@ const Treasury = (props: TreasuryProps) => {
   return (
     <div className={classNames(styles.treasury, className)}>
       <p
-        className={`${globalStyles.secondaryColor} ${globalStyles.textSmall} ${globalStyles.uppercase} ${globalStyles.textRight}`}
+        className={`${styles.label} ${globalStyles.secondaryColor} ${globalStyles.textSmall} ${globalStyles.uppercase} ${globalStyles.textRight}`}
```

Comment:
```suggestion
        className={classNames(styles.label, globalStyles.secondaryColor, globalStyles.textSmall, globalStyles.uppercase, globalStyles.textRight)}
```

Reply:
I am planning on resolving these in one go. This is tracked in https://github.com/api3dao/api3-dao-dashboard/issues/146

But I don't wanna introduce rebase issues by doing it now.